Add Missing Fifth Argument to mail() Function Call

miqrogroove opened the pull request:
This is desperately needed to fix Issue #5190.  In case of any unexpected problem, the admin can use

`$a->config['system']['sendmail_params'] = false;`

MrPetovan commented:
Can you please add this new htconfig setting to `doc/htconfig.md` ?

MrPetovan commented on src/Util/Emailer.php:
```diff
@@ -72,20 +72,28 @@ public static function send($params)
 		$multipartMessageBody .=
 			"--" . $mimeBoundary . "--\n";					// message ending
 
+		if (Config::get("system", "sendmail_params", true)) {
```
Can you please add this new htconfig setting to `doc/htconfig.md` ?

MrPetovan replied:
I would have preferred it to be tied to another settings where you can select your email provider but we don't have that at the moment, so this is low impact and fine as Sendmail is the default on many system, I believe.

MrPetovan commented on doc/htconfig.md:
```diff
@@ -85,6 +85,7 @@ Example: To set the automatic database cleanup process add this line to your .ht
 * **pushpoll_frequency** -
 * **qsearch_limit** - Default value is 100.
 * **remove_multiplicated_lines** (Boolean) - If enabled, multiple linefeeds in items are stripped to a single one.
+* **sendmail_params** (Boolean) - Normal sendmail command parameters will be added when the PHP mail() function is called for sending e-mails.  This ensures the Sender Email address setting is applied to the message envelope rather than the host's default address.
```
Please add the default value to this line and the disabling suggestion if not using Sendmail.

miqrogroove commented:
Updated htconfig.md.  Fixed an error that came up in the first commit.

Tested and working now on my live server.  Also tested the new htconfig option and confirmed it restored the old (unwanted) mail behavior.

MrPetovan commented:
Fixes #5190
